### PR TITLE
DOC-5612: Update crypto docs

### PR DESCRIPTION
**Pull request by amiegrace12**
Preview here: https://amie.preview.docs.r3.com/en/platform/corda/5.0/deploying-operating/observability/metrics/crypto-worker.html
https://amie.preview.docs.r3.com/en/platform/corda/5.0/developing-applications/api/application/crypto.html
https://amie.preview.docs.r3.com/en/platform/corda/5.0/reference/corda-cli/initial-config.html#create-crypto-config

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

**Review by dickon — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/crypto-worker.md [hunk 1]
```diff
@@ -13,32 +13,21 @@ section_menu: corda5
...
     On the crypto worker side, the crypto worker attempts to find the key in the keys hosted for the virtual node that
     sent the request and if found, it signs the bytes and returns the signature.
-   The metrics in the following table pertain to the time taken to handle the entire
-    `sign` request. Additionally, there are more detailed metrics related to key caches and the 'sign' operation itself.
-  * `SigningService.findMySigningKeys`: The `findMySigningKeys` operation sends a set of keys to the crypto worker,
-    which then filters and returns the keys owned by the calling virtual node.
-    The metrics in the following table are related to the time taken to
-    handle the entire `findMySigningKeys` request. Additionally, there are metrics related to key caches.
+    The metrics in the following table pertain to the time taken to handle the entire `sign` request. Additionally, there are more detailed metrics related to key caches and the 'sign' operation itself.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/crypto-worker.md:26]:
```suggestion
    The metrics in the following table pertain to the time taken to handle the entire `sign` request. Additionally, there are more detailed metrics related to key caches and the `sign` operation itself.
```

**Review by yarivus — COMMENTED** [review]
thx @amiegrace12

**Pull request merged**